docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Create Docker image that can be used for testing Zephyr network
+# sample applications.
+
+FROM fedora
```

Comment:
Any idea of using a more light-weight base? (I actually not sure how heavy-weight is that, but I know that people who want to optimize things, use lite-weight bases like alpine linux or something).

Reply:
So far it is useful to have Fedora as a base since it provides a familiar base distro with needed development libraries and headers and where it is easy to add other net applications needed for testing. I would not like to do premature size and tool optimization before we have the set of tests and tools somewhere far beyond the initial echo server. As it is now, the only heavy-weight issue is the size of the image, the tools will be as fast on this image as on any other we come up with. So the ease of use and development wins over image size issues for the time being.